changelog.txt

```diff
@@ -16,6 +16,8 @@ that repo.
 ## New Scripts
 
 ## Fixes
+- `gui/unit-syndromes`: allow the window widgets to be interacted with
```

Comment:
this looks like a merge error -- this line is copied from the last version's release notes